tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "outDir": "lib",
     "rootDir": "src",
     "sourceMap": true,
+    "target": "ES6",
```

Comment:
@tancredi you might want to experiment with setting this to ES2017 or newer since you already support only Node.js >= 10.

Reply:
Sure, but one thing at a time ;) this solves a problem and is good for now

Reply:
I know, just wanted to note this so that you have a look at later